main.tex

```diff
@@ -406,7 +406,11 @@ \subsection*{Departures from Yeadon's work}
       denominator. In this scenario, Yeadon still employs the formulae for
       stadium solids but sets the thickness to be very
       small~\cite{Yeadon1990f}.  Instead, we manipulate the equations so that
-      the approximation is not necessary.
+      the approximation is not necessary. In the case that only one of the
+      stadia for a solid has zero thickness, the divide-by-zero issue is
+      removed by computing the mass properties for the stadium solid in which
+      the two stadia are swapped. However, this manipulation does not work if
+      both stadia have zero thickness; for this case, we refer you to the code.
```

Comment:
It's okay to have zero-thickness. For example, both stadia having zero thickness is a truncated cone. We should support that. So I don't think we should throw an error.

The description of the manipulations is here: https://github.com/chrisdembia/yeadon/blob/master/yeadon/solid.py#L397

I'd have to give a lot of background in the paper to actually describe the manipulation.

I'll clarify that the degeneracy is fine to have though.